### DISCUSSION TITLE
Run mountain duck without admin privilege on Windows

### DISCUSSION THREAD

**Discussion by qubist-pixel-ux** [iterate-ch/cyberduck#16332] [Q&A] [answered]
How can I mount folder for Standard (non admin) account ? I have admin access but the standard account user does not. The idea is to open mountain duck when required by the user (not keep it running in background).

**Comment by AliveDevil** [comment]
Mountain Duck requires administrator privileges when installed only.
After installation any user can start and connect any drive in their own session.

**Comment by qubist-pixel-ux** [comment]
In my case, it always asks for Administrator privileges. The account was Administrator account before it was changed to standard account.
________________________________

**Reply by qubist-pixel-ux** [reply, marked as answer]
Nevermind, it was because, "Run as Administrator" was checked in compatibility mode.
